Kernel/Term.hpp

```diff
@@ -905,6 +915,8 @@ class alignas(8) Term
   unsigned _isTwoVarEquality : 1;
   /** Weight of the symbol */
   unsigned _weight;
+  /** Cached weight of the term for KBO, otherwise -1 and invalid. */
+  int _kboWeight;
```

Comment:
Could a comment be made on the difference between `weight` and `kboWeight`? I always find this confusing.